app/v1/services/CreateAndAmendReliefInvestmentsService.scala

```diff
@@ -40,19 +39,25 @@ class CreateAndAmendReliefInvestmentsService @Inject() (connector: CreateAndAmen
       logContext: EndpointLogContext,
       correlationId: String): Future[ServiceOutcome[Unit]] = {
 
-    val result = for {
-      responseWrapper <- EitherT(connector.amend(request)).leftMap(mapDownstreamErrors(desErrorMap))
-    } yield responseWrapper
-
-    result.value
+    connector.amend(request).map(_.leftMap(mapDownstreamErrors(downstreamErrorMap)))
   }
 
-  private def desErrorMap =
-    Map(
+  private def downstreamErrorMap = {
+    val errors = Map(
       "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
-      "FORMAT_TAX_YEAR"           -> TaxYearFormatError,
+      "INVALID_TAX_YEAR"          -> TaxYearFormatError,
+      "INVALID_PAYLOAD"           -> InternalError,
+      "UNPROCESSABLE_ENTITY"      -> InternalError,
       "SERVER_ERROR"              -> InternalError,
       "SERVICE_UNAVAILABLE"       -> InternalError
```

Comment:
I think the old tech spec may have missed out an INVALID_CORRELATIONID error (i..e no second underscore _). The downstream spec for #1629 does include a INVALID_CORRELATIONID error so probs best to add a mapping for it here and update the old spec too
<img width="721" alt="image" src="https://user-images.githubusercontent.com/106095575/208408931-35af21ce-2f2c-42b0-9781-64b6fd058d49.png">

Reply:
Good spot!